aws/ecr/ecr.tf

```
@@ -108,3 +108,14 @@ resource "aws_ecr_repository" "system_status" {
     scan_on_push = true
   }
 }
+
+resource "aws_ecr_repository" "github_arc_runner" {
```

```suggestion
resource "aws_ecr_repository" "github_Art_runner" {
```
Ok just kidding there

---

More seriously though, could we name this as a repository rather than a runner?
```suggestion
resource "aws_ecr_repository" "github_arc_repo" {
```